legacy/src/app/bootstrap.js

```diff
@@ -70,7 +70,7 @@ const bootstrapOverWebsocket = config => {
         const requiredConfigKeys = [
           "completed_intro",
           "maas_name",
-          "maas_uuid",
+          "uuid",
```

Comment:
In entry.js there are a couple of references to `$window.CONFIG.maas_uuid` which will need to be updated.